src/integration/groovy/pl/allegro/tech/build/axion/release/ExtendingTasksIntegrationTest.groovy

```diff
@@ -34,7 +34,7 @@ class ExtendingTasksIntegrationTest extends BaseIntegrationTest {
         task << [
             'CreateReleaseTask',
             'MarkNextVersionTask',
-            'OutputCurrentVersionTask',
+            // 'OutputCurrentVersionTask', FIXME causes test to fail - Type 'pl.allegro.tech.build.axion.release.OutputCurrentVersionTask' property 'versionConfig.versionIncrementer.creator' is missing an input or output annotation
```

Comment:
what about that? :)

Reply:
As I dmed you - this is the test that uncovers some missing Gradle task annotations (`@Input`, `@Output` etc.). 
I am not familiar with all the plugin domain logic so I left this out intentionally.
I saw that you added those annotations to other parts of the code so would you mind looking into this and adding appropriate annotations?

as stated in the error message - `creator` is not annotated but it's a local variable:

```groovy
    static Closure<Version> versionIncrementerFor(String name, Map configuration = [:]) {
        PredefinedVersionIncrementer creator = values().find { it.name == name }
        if (creator == null) {
            throw new IllegalArgumentException("There is no predefined version incrementer with $name name. " +
                "You can choose from: ${values().collect { it.name }}")
        }
        return { VersionIncrementerContext context -> creator.versionIncrementer(context, configuration) }
    }
```

Reply:
Ok, I got away with simpler solution - instead of calling `versionIncrementerFor` I used direct enum value